src/types/react-table-config.ts

```diff
@@ -106,9 +107,24 @@ declare module 'react-table' {
     // extends UseGlobalFiltersColumnOptions<D>,
     // UseGroupByColumnOptions<D>,
     // UseResizeColumnsColumnOptions<D>,
-    extends UseSortByColumnOptions<D> {
+    extends UseSortByColumnOptions<D>,
+      UseFiltersColumnOptions<D> {
     columnClassName?: string;
     cellClassName?: string;
+    /**
+     * Type of the data in cell. Used for manual filtering.
+     */
+    fieldType?: FieldType;
+    /**
+     * Filter component used as a column filter.
+     */
+    Filter?: Renderer<FilterProps<D>>;
+    /**
+     * String value or custom function to use for filtering.
+     * Possible string values: `text`, `exactText`, `exactTextCase`, `includes`, `includesAll`, `exact`, `equals`, `between`.
+     * More info about these filters: https://github.com/tannerlinsley/react-table/blob/master/src/filterTypes.js
+     */
+    filter?: FilterType<D> | DefaultFilterTypes | string;
```

Comment:
Are these `Filter` vs `filter` prop names provided by react-table? Because it's very confusing (I would expect `filter` and `filterFunction`)

Reply:
Yeah these come from `react-table`, I just added comments.